### ISSUE TITLE
Cannot find module '.../is-plain-object.js'

### ISSUE THREAD

**Issue by The-Compiler** [Gottox/irc-message-action#5]
Since the recent update (I think), this action always seems to fail for me:

```
internal/modules/cjs/loader.js:305
      throw err;
      ^

Error: Cannot find module '/home/runner/work/_actions/Gottox/irc-message-action/v1/node_modules/is-plain-object/dist/is-plain-object.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:297:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:528:18)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:784:27)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/runner/work/_actions/Gottox/irc-message-action/v1/node_modules/@octokit/endpoint/dist-node/index.js:5:21)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32) {
  code: 'MODULE_NOT_FOUND',
  path: '/home/runner/work/_actions/Gottox/irc-message-action/v1/node_modules/is-plain-object/package.json',
  requestPath: 'is-plain-object'
}
```

**Comment by dR3b** [comment]
@Gottox 
Please update README.md!

**Comment by Gottox** [comment]
fixed in 1.3.0. Sorry for the delay.
